builtin/audit/file/backend.go

```diff
@@ -18,9 +18,9 @@ func Factory(conf *audit.BackendConfig) (audit.Backend, error) {
 		return nil, fmt.Errorf("nil salt")
 	}
 
-	path, ok := conf.Config["path"]
+	path, ok := conf.Config["file_path"]
```

Comment:
If `file_path` is empty, for now, we should still look for `path`. This keeps it backwards compatible with API clients that are not yet at 0.5.2. If it were 0.6 I'd worry less but I think people aren't expecting breaking API changes in the middle of the 0.5.x series. We did some in 0.5.1, but for enhanced security, rather than tidying up.